### ISSUE TITLE
data-table thead does not support  style settings

### ISSUE THREAD

**Issue by zhuyifang** [quasarframework/quasar#907]
data-table td support style settings, which is great
But thead does not support, then in a lot of time is very inconvenient, it is recommended to provide thead in the td style control # @lennonjesus

**Comment by rstoenescu** [comment]
QDataTable is running a revamp as we speak. Will be available in v0.15.
You'll be able to configure it to the core.

Thanks for your patience!

v0.15 roadmap: http://forum.quasar-framework.org/topic/815/quasar-v0-15-roadmap

**Comment by zhuyifang** [comment]
It's fast and cool.
I would like to make a few suggestions:
1. Do not need to provide too many components
We found in the use of the process, many components of the function is too simple, so it seems redundant. Such as dialog box components, only need to provide a set of modal-based html structure on it, the user can expand their own.
2. Function naming in the component
Naming should follow the user's habits, bootstarp has a lot of people use, so for the Secondary Success Danger Warning Info Light Dark these have been very adapted, but quasar but their own definition of a set: such as Toast.create.positive ({ ...}), Toast.create.negative ({...}), Toast.create.warning ({...}), Toast.create.info ({...}).
3. As much as possible using css3 animation
Because I found that the animation in the Stepper component is based on changing the height implementation of dom, similar to the early jquery


My English is very bad, use google translation, make do with it
